support/mender-inventory-geo

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+#
+# Example script called by Mender client to collect inventory data for a
+# particular device. The script needs to be located in $(datadir)/mender and its
+# name shall start with `mender-inventory-` prefix. The script shall exit with
+# non-0 status on errors. In this case the agent will discard any output the
+# script may have produced.
+#
+# The script shall output inventory data in <key>=<value> format, one entry per
+# line. Entries appearing multiple times will be joined in a list under the same
+# key.
+#
+# $ ./mender-inventory-geo
+# country=Poland
+# city=Krakow
+# latitude=50.05750
+# longitude=19.98020
+# ipv4=127.0.0.1
+#
+# The example script collects information on geo localization
+# this can be done in one line:
+# curl https://ipvigilante.com/`curl -4 ifconfig.io 2>/dev/null` 2>/dev/null | jq -r 'select(.status == "success") | "country=" + .data.country_name, "city="+.data.city_name, "latitude="+.data.latitude, "longitude="+.data.longitude, "ipv4="+.data.ipv4 | @text'
+# but in the following we assume busybox version of the utilities.
+# see below for the configuration of the attributes names
+
+function err() {
+ local rc=$1
+
+ shift
+ echo "$0 $* " >&2
```

Comment:
Maybe:
echo "$0`:` $*" >&2
Also, there is no need for a space after the `$*` I think.

Reply:
sure.